docs/advanced/monitoring-unhandled-actions.md

```diff
@@ -24,42 +42,51 @@ Setting `warnOnUnhandledActions` to a truthy value will tell the logger to warn
 We can ignore specific actions that should not be logged if they have never been handled. For instance, if we're using the `@ngxs/router-plugin` and don't care about router actions like `RouterNavigation`, then we may add it to the `ignore` array:
 
 ```ts
-import { NgxsDevelopmentModule } from '@ngxs/store';
+import { provideStore, withNgxsDevelopmentOptions } from '@ngxs/store';
 import { RouterNavigation, RouterCancel } from '@ngxs/router-plugin';
 
-@NgModule({
-  imports: [
-    NgxsDevelopmentModule.forRoot({
-      warnOnUnhandledActions: {
-        ignore: [RouterNavigation, RouterCancel]
-      }
-    })
+export const appConfig: ApplicationConfig = {
+  providers: [
+    provideStore(
+      [],
+      withNgxsDevelopmentOptions({
+        warnOnUnhandledActions: true
+      })
+    )
   ]
-})
-export class AppModule {}
+};
 ```
 
 > 💡 It's best to import this module only in development mode. This may be achieved using environment imports. See [dynamic plugins](../recipes/dynamic-plugins.md).
 
 Ignored actions can be also expanded in lazy modules. The `@ngxs/store` exposes the `NgxsUnhandledActionsLogger` for these purposes:
 
 ```ts
-import { Injector } from '@angular/core';
+import { inject } from '@angular/core';
 import { NgxsUnhandledActionsLogger } from '@ngxs/store';
 
 declare const ngDevMode: boolean;
 
-@NgModule({
-  imports: [NgxsModule.forFeature([LazyState])]
-})
-export class LazyModule {
-  constructor(injector: Injector) {
-    if (ngDevMode) {
-      const unhandledActionsLogger = injector.get(NgxsUnhandledActionsLogger);
+const providers = [provideStates([LazyState])];
+
+if (ngDevMode) {
+  providers.push({
+    provide: ENVIRONMENT_INITIALIZER,
+    multi: true,
+    useValue: () => {
+      const unhandledActionsLogger = inject(NgxsUnhandledActionsLogger);
       unhandledActionsLogger.ignoreActions(LazyAction);
     }
-  }
+  });
```

Comment:
Unrelated to this PR, do you think we could provide a more convenient way of doing this type of thing?